.github/workflows/nightly_wheels_build_and_test.yml

```diff
@@ -0,0 +1,47 @@
+name: Nightly Wheels Build and Test
```

Comment:
Please note that putting this into an separate file will create another separate build (we have max 5 in parallel)
=> Each file counts as one workflow (i.e. CI, nightly and this too)
In the future if we want to include it in the regular CI then it might be moved to the `ci.yml`

Reply:
Ah, than i misunderstood you. I can simply put this job into the nightly file. We can then still move it to the regular ci file if wanted in the future.